interface/wx/protocol/http.h

```diff
@@ -12,6 +12,9 @@
 
     wxHTTP can thus be used to create a (basic) HTTP @b client.
 
+    @note If you want HTTPS, IPv6, Proxy detection, authentication, etc.
+        support consider using wxWebRequest instead.
```

Comment:
I'd be slightly more forceful and say
```suggestion
    @note In practice, for any but the most trivial cases, e.g. if you need HTTPS, HTTP/2 or IPv6,
            proxy detection, authentication, etc. support please use wxWebRequest instead.
```